app/src/main/res/values/strings.xml

```diff
@@ -164,20 +108,18 @@
 
     <string name="connection_info">Connection info</string>
     <string name="status_title">Wi-Fi info</string>
-    <string name="status_ip_4">IPV4: %s</string>
+    <string name="status_ip_4">IPv4: %s</string>
```

Comment:
I think Banshi wanted IPV4

Reply:
No, you had IpV4 before. The proper capitalization is: https://pl.wikipedia.org/wiki/IPv4

Reply:
There was IPV4 before. It's visible on diff. 
From Banshi: "Change IpV4 to IPV4".
I don't mind both, just informing you.